pages/games/[chainLabel]/[gameId].tsx

```diff
@@ -255,18 +257,16 @@ function GamePage({
                   <NetworkDisplay chainId={chainId} />
                 </HStack>
               </Link>
-              {isOwner ||
-                isAdmin ||
-                (isMaster && (
-                  <Button
-                    onClick={() =>
-                      openActionModal(GameMasterActions.UPDATE_GAME_METADATA)
-                    }
-                    size="sm"
-                  >
-                    edit
-                  </Button>
-                ))}
+              {(isOwner || isAdmin || isMaster) && (
```

Comment:
We need to change this to `onlyAdmin` as per contract.

Reply:
Changed in e4db133ce344c2ff0a051780d432e17708d779eb